genai-perf/genai_perf/wrapper.py

```diff
@@ -66,7 +65,8 @@ def add_inference_load_args(args: Namespace) -> List[str]:
     @staticmethod
     def add_payload_args(args: Namespace) -> List[str]:
         cmd = []
-        cmd += ["--fixed-schedule"]
+        if args.prompt_source == PromptSource.PAYLOAD:
+            cmd += ["--fixed-schedule"]
```

Comment:
Kind of a semantic question, but does the existence of `payload:...` in `--input-file` inherently imply that there will be payloads containing the `timestamp` field?

Because it seems more direct/accurate to say that `--fixed-schedule` should be added to the PA command when `timestamp` field is included in all of the payloads. Not simply just the existence of payloads.

Am I misunderstanding something?

Reply:
@lkomali @debermudez @nv-hwoo This was what I was asking about earlier. It does seem that it'd be impossible to use the payloads for multi-turn with this logic. There could be a hacky workaround, but it'd be ideal to find a better way to do this before the feature is delivered.

I don't think we want to block the merging of this onto main, but it looks like we actually do need to prioritize some form of detection of fixed schedule vs multi-turn for this feature.

Reply:
Thanks for identifying this, Matt.

Reply:
@dyastremsky @lkomali I see the PR was merged, but I didn't feel that this conversation was resolved.

Can we get more of a response here to my feedback?